Core/Tests/Speckle.Core.Tests.Integration/Api/GraphQL/Resources/ProjectResourceExceptionalTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Speckle.Core.Api;
+using Speckle.Core.Api.GraphQL;
+using Speckle.Core.Api.GraphQL.Enums;
+using Speckle.Core.Api.GraphQL.Inputs;
+using Speckle.Core.Api.GraphQL.Models;
+using Speckle.Core.Api.GraphQL.Resources;
+
+namespace Speckle.Core.Tests.Integration.API.GraphQL.Resources;
+
+[TestOf(typeof(ProjectResource))]
+public class ProjectResourceExceptionalTests
+{
+  private Client _testUser,
+    _secondUser,
+    _unauthedUser;
+  private Project _testProject;
+  private ProjectResource Sut => _testUser.Project;
+
+  [OneTimeSetUp]
+  public async Task Setup()
+  {
+    _testUser = await Fixtures.SeedUserWithClient();
+    _secondUser = await Fixtures.SeedUserWithClient();
+    _unauthedUser = Fixtures.Unauthed;
+    _testProject = await _testUser.Project.Create(new("test project123", "desc", null));
+  }
+
+  //TODO: There's got to be a smarter way to parametrise this.
```

Comment:
I'd rather we have POC if we need to do something before launch, let's NOT have TODOs but prefer jira tickets

Reply:
I'll figure out how else to express this comment
I intended this comment to be a suggestion to a future dev incase anyone feels bold enough to re-write these tests, not as a statement that they should be rewritten...